symbol.go

```diff
@@ -0,0 +1,15 @@
+package gore
+
+import (
+	"errors"
+)
+
+var ErrSymbolNotFound = errors.New("symbol not found")
+
+// Symbol A primitive representation of a symbol.
```

Comment:
Symbol is a generic representation of [debug/elf.Symbol], [debug/pe.Symbol], and [debug/macho.Symbol].